test-data/unit/cmdline-based-baseline.test

```diff
@@ -575,3 +576,28 @@ def f(): ...
 [out]
 a.py:2:1: error: Unexpected keyword argument "a" for "f"  [call-arg]
 b.py:4:1: note: "f" defined here
+
+
+[case testErrorLink]
+# cmd: mypy a.py --show-error-code-links
+[file a.py]
+a
+[file .mypy/baseline.json]
+{
+    "files": {
+        "a.py": [
+            {
+                "code": "name-defined",
+                "column": 0,
+                "message": "Name \"a\" is not defined",
+                "offset": 1,
+                "src": "a",
+                "target": "a"
+            }
+        ]
+    },
+    "format": "1.7",
+    "targets": [
+        "file:a.py"
+    ]
+}
```

Comment:
wheres the error link?

Reply:
it's testing that the error link doesn't appear if the error is baselined, I've added a comment